src/test/lib/storage/storage_manager_test.cpp

```diff
@@ -2,13 +2,15 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <numeric>
 
 #include "base_test.hpp"
 
 #include "hyrise.hpp"
 #include "logical_query_plan/stored_table_node.hpp"
 #include "storage/table.hpp"
 #include "utils/meta_table_manager.hpp"
+#include "./storage_manager_test_util.cpp"
```

Comment:
```suggestion
#include "storage_manager_test_util.cpp"
```
Maybe something else but don't use `./`.